ipamutils/utils.go

```diff
@@ -48,3 +62,75 @@ func initGranularPredefinedNetworks() []*net.IPNet {
 	}
 	return pl
 }
+
+// InitAddressPools allows to initialize the local and global scope predefined
+// address pools to the desired values. It fails is invalid input is passed
+// or if the predefined pools were already initialized.
+func InitAddressPools(list []*PredefinedPools) error {
+	var (
```

Comment:
I don't think you really need up-front C-style declarations like these. In Go you can just directly assign